modules/manually-removing-cloud-creds.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
```

Comment:
What precisely is this adoc file documenting? It is titled as if it were going to document removing the root Secret with the admin-level cloud credentials. But the content instructs
1) remove the root Secret
2) Identify and remove the individual IAM Users created by CCO in mint mode.
3) Delete the Secrets holding the minted credentials for each CredentialsRequest CR

After step 2/3, we are left in a bad place because we've erased the user/credentials that a component is relying on, but in step 1 we took away the ability for CCO to fix/repair.

If this doc is about removing the root secret, then nothing beyond step 1 is needed.

If this doc is about manually rotating credentials, then step 1 should be something like:
If you have removed the adming-level Secret from the kube-system namespace, restore the Secret with appropriate admin-level credentials so that CCO can re-mint new IAM users and credentials.

Reply:
Thanks, @joelddiaz -
Ok, I think perhaps I am mentally conflating complete removal (i.e., I want to run in manual mode or something) with just removing the cred used to mint but **not** the minted creds. Lemme go untangle that, but if you think based on this reply that I'm **still** missing something please reach out :)
Cc: @akhil-rane